sync-debug.js

```diff
@@ -115,7 +113,10 @@ const performLivepush = _.debounce(async (livepush, containerId, docker) => {
 		console.log('SYNC: executing:', command);
 	});
 	livepush.on('commandOutput', ({ output }) => {
-		console.log(`\t${output.data.toString()}`);
+		const message = output.toString().trim();
```

Comment:
There's no `toString` method on `CommandOutput`

Reply:
Actually there is but it will output `'[object Object]'`

Reply:
Fixed.